main.rb

```diff
@@ -3,170 +3,60 @@
 require_relative('./book')
 require_relative('./teacher')
 require_relative('./rental')
-
-@books = []
-@people = []
-def list_book
-  if @books.length.zero?
-    puts 'Book list is empty, Please add book'
-
-  else
-    @books.each_with_index do |book, index|
-      puts "[Book] #{index + 1} - Title: #{book.title}, Author: #{book.author}"
-    end
-  end
-end
-
-def list_people
-  if @people.length.zero?
-    puts "There's no person, Please add a person first"
-    main_method
-  else
-    @people.each_with_index do |person, index|
-      if person.is_a?(Teacher)
-        puts "[Teacher] #{index + 1} - Name: #{person.name}, Age: #{person.age}"
-      else
-        puts "[Student] #{index + 1} - Name: #{person.name}, Age: #{person.age}"
-      end
-    end
+require_relative('./create')
+require_relative('./display')
+class App
+  def initialize
+    @people = []
+    @books = []
   end
```

Comment:
Please extract the `App` class from the `main.rb` file and place it in a separate file. This way you will be following Single Responsibility Rule. Kindly read more about SRA [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid).